src/pydicom/_private_dict.py

```diff
@@ -1473,6 +1473,7 @@
         '01F1xx0D': ('DS', '1', 'Rotation Angle', ''),  # noqa
         '01F1xx0E': ('FL', '1', 'Unknown', ''),  # noqa
         '01F1xx26': ('DS', '1', 'Pitch', ''),  # noqa
+        '01F11026': ('FD', '1', 'Pitch', ''),  # noqa
```

Comment:
Maybe I missed something, but shouldn't this go in the 01F1xx26 line rather than being fixed to xx=10?

Reply:
I figured that there might be older elements that still use xx26 with DS. I tested it and the exact match has priority.

Reply:
Also, this PR kind of negates the need for #1560, do you still think that PR is worth it?

Reply:
> Also, this PR kind of negates the need for #1560, do you think that PR is still worthwhile?

Hmmm... reviewed that a fair bit and I'd like to agree it negates, but I'm having trouble wrapping my brain around it.  It seems that one difference is that #1560 automatically applied to all elements as they are converted, this one by itself requires user code to know what needs to be changed, even nested elements.

Perhaps the two can be bridged, as long as the existing callback mechanism (prior to the Hooks in #1560) exists, anyone can do anything they want in a conversion, but that kind of brings it back to the original objection in #1556 of wanting to use part of the existing pydicom machinery rather than replacing everything.

So, ... in summary, I don't know ... this is certainly much much simpler, #1560 is getting quite complex, IMO.

Reply:
I think you're right, #1560 is probably more useful where systematic intervention is needed, whereas this is more useful for the occasional odd fix.